error[E0308]: mismatched types

error[E0308]: mismatched types
   --> src\mesh_material\mod.rs:242:18
    |
242 |             min: self.position - self.radius,
    |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `glam::f32::vec3::Vec3`, found struct `bevy::prelude::Vec3`
    |
    = note: struct `bevy::prelude::Vec3` and struct `glam::f32::vec3::Vec3` have similar names, but are actually distinct types
note: struct `bevy::prelude::Vec3` is defined in crate `glam`

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -18,7 +18,7 @@ warn_mesh_load = []
 [dependencies]
 bytemuck = "1.9"
 itertools = "0.10"
-bvh = "0.7.1"
+bvh = "= 0.7.1"
 bitflags = "1.3"
 serde = "1.0"
 serde_variant = "0.1.1"

